packages/hoprd/crates/hoprd-hoprd/Cargo.toml

```diff
@@ -1,32 +1,35 @@
 [package]
-name = "hoprd-hoprd"
-version = "0.2.0"
+name = "hoprd"
+version = "2.1.0"
 authors = ["HOPR Association <tech@hoprnet.org>"]
 edition = "2021"
 description = "Contains the main entry point of HOPR daemon application"
 homepage = "https://hoprnet.org/"
 repository = "https://github.com/hoprnet/hoprnet"
 license = "GPL-3.0-only"
 
-# TODO: change this to a binary crate once the migration is complete
-[lib]
-crate-type = ["cdylib", "rlib"]
-
 [dependencies]
+async-std = { workspace = true, features = ["attributes"] }
+tokio = { version = "1.33", default-features = false }
```

Comment:
Leftover?

Reply:
Yes... from the HEAD~1 commit, ai wanna setup a dependency prining utility after migrating